lib/lotus/commands/generate.rb

```diff
@@ -57,6 +59,13 @@ def spec_root
         @spec_root ||= Pathname.new('spec')
       end
 
+      # @since 0.3.0
+      # @api private
+      def model_root
+        @model_root ||= Pathname.new(['lib', target.to_s.split(::File::SEPARATOR).last]
```

Comment:
Why do split and get the last part of target path? as of `['lib', target.to_s.split(::File::SEPARATOR).last]`?